was moved markup and styles to corresponding components

AndreyBerezyanskiy opened the pull request:
[DEMO LINK](https://AndreyBerezyanskiy.github.io/react_static-list-of-posts/)

GarryRocksman commented on src/components/PostInfo/PostInfo.tsx:
```diff
@@ -1,5 +1,91 @@
+
+            <a
+              className="CommentInfo__email"
+              href="mailto:Telly_Lynch@karl.co.uk"
+            >
+              Telly_Lynch@karl.co.uk
+            </a>
+          </div>
+
+          <div className="CommentInfo__body">
+            dolorum voluptas laboriosam quisquam ab totam beatae et aut
+            aliquid optio assumenda voluptas velit itaque quidem voluptatem
+            tempore cupiditate in itaque sit molestiae minus dolores magni
+          </div>
+        </div>
+
+        <div className="CommentInfo">
+          <div className="CommentInfo__title">
+            <strong className="CommentInfo__name">
+              odio adipisci rerum aut animi
+            </strong>
+
+            {' by '}
+
+            <a
+              className="CommentInfo__email"
+              href="mailto:Nikita@garfield.biz"
+            >
+              Nikita@garfield.biz
+            </a>
+          </div>
+
+          <div className="CommentInfo__body">
+            quia molestiae reprehenderit quasi aspernatur aut expedita
+            occaecati aliquam eveniet laudantium omnis quibusdam delectus
+            saepe quia accusamus maiores nam est cum et ducimus et vero
+            voluptates excepturi deleniti ratione
+          </div>
+        </div>
+      </div> */}
```
delete commented code

AndreyBerezyanskiy replied:
thank you a lot!

GarryRocksman reviewed (approved):
GJ 🔥

alexandra-protyanova commented on src/App.tsx:
```diff
@@ -1,109 +1,38 @@
 
-          <p>
-            {' Posted by  '}
+function getCommentsById(postId: number): Comment[] {
```
you can return array immediately, by set `()` without `return`

alexandra-protyanova replied:
I think we should do this)

alexandra-protyanova commented on src/components/PostList/PostList.tsx:
```diff
@@ -1,5 +1,16 @@
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo key={post.id} post={post} />
+    ))}
+  </div>
 );
```
```suggestion
export const PostList: React.FC<Props> = ({ posts }) => (
  <div className="PostList">
    {posts.map(post => (
      <PostInfo key={post.id} post={post} />
    ))}
  </div>
);
```

alexandra-protyanova commented on src/components/PostInfo/PostInfo.tsx:
```diff
@@ -1,5 +1,43 @@
+          {title}
+        </h3>
+
+        <p>
```
in case of missing user you will render `Posted by` anyway, I don't think it is a good idea

alexandra-protyanova commented on src/components/CommentList/CommentList.tsx:
```diff
@@ -1,5 +1,25 @@
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => {
+  if (comments.length === 0) {
+    return (
+      <b data-cy="NoCommentsMessage">No comments yet</b>
+    );
+  }
```
u can use ternary instead of `if`

AndreyBerezyanskiy replied:
done. Thanks)

alexandra-protyanova commented on src/components/PostList/PostList.tsx:
```diff
@@ -1,5 +1,16 @@
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+
+  <div className="PostList">
+    {posts.map(post => (
+      <PostInfo key={post.id} post={post} />
+    ))}
+  </div>
```
```suggestion
export const PostList: React.FC<Props> = ({ posts }) => (
  <div className="PostList">
    {posts.map(post => (
      <PostInfo key={post.id} post={post} />
    ))}
  </div>
```

alexandra-protyanova reviewed (changes requested):
There isn't any changes since last review, I think you forgot to commit changes

AndreyBerezyanskiy commented:
> There isn't any changes since last review, I think you forgot to commit changes

Something's gone wrong, because I committed it yesterday. I've fixed. Check it out, please)

alexandra-protyanova commented on src/components/CommentList/CommentList.tsx:
```diff
@@ -1,5 +1,21 @@
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  comments.length === 0
+    ? (<b data-cy="NoCommentsMessage">No comments yet</b>)
+    : (
+      <div className="CommentList">
+        {comments.map(comment => (
+          <CommentInfo key={comment.id} comment={comment} />
+        ))}
+      </div>
+    )
```
```suggestion
  !comments.length
    ? (<b data-cy="NoCommentsMessage">No comments yet</b>)
    : (
      <div className="CommentList">
        {comments.map(comment => (
          <CommentInfo key={comment.id} comment={comment} />
        ))}
      </div>
    )
```

alexandra-protyanova reviewed (approved):
GJ ⭐️
